frontend/doc-site/032__Contribute and Publish Data.mdx

```diff
@@ -0,0 +1,56 @@
+# Contributing Data
+
+Cellxgene has a rapidly growing single cell data corpus because of generous contributions from researchers like you! If you feel like you have a single cell dataset that meets our requirments (see below), then feel free to reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com).
+
+## Publishing Process
+
+The general process for submission to the portal looks like this:
+- reach out to our curation team at [cellxgene@chanzuckerberg.com](cellxgene@chanzuckerberg.com) with a description of the dataset that you'd like to contribute. Generally speaking this would be derived from either human or mouse. We not allow submission of data sets from:
+  - organisms other than human or mouse
+  - cell lines
+  - drug screens 
+- recieve confirmation form the curation team will accept your dataset along with instructions on how to format your dataset for submission.
```

Comment:
RE _recieve confirmation form the curation team will_ - 

1. recieve -> receive 
2. form -> from ?

Is this intended to be something like _If your request is approved, the curation team will respond with further instructions on how to format ...._

Reply:
edited the pr to address both of these comments